Engine/src/Engine/Memory/StackAllocator.cpp

```diff
@@ -45,7 +41,6 @@ void* StackAllocator::Alloc(const Size size, const U8 alignment) {
 	}
 
 	m_top = newTop;
-
 	return reinterpret_cast<void*>(alignedAddress);
```

Comment:
does it need to be reinterpret cast here?

Reply:
Yes, as static cast may change the data